packages/contracts/contracts/StabilityPool.sol

```diff
@@ -365,82 +353,35 @@ contract StabilityPool is LiquityBase, StabilityPoolStorage, CheckContract, ISta
+         This lets the event log make more sense (i.e. so it appears that first the SOV gain is withdrawn
         and then it is deposited into the Trove, not the other way around). */
-        emit ETHGainWithdrawn(msg.sender, depositorETHGain, ZUSDLoss);
+        emit SOVGainWithdrawn(msg.sender, depositorSOVGain, ZUSDLoss);
         emit UserDepositChanged(msg.sender, compoundedZUSDDeposit);
 
-        ETH = ETH.sub(depositorETHGain);
-        emit StabilityPoolETHBalanceUpdated(ETH);
-        emit EtherSent(msg.sender, depositorETHGain);
-
-        borrowerOperations.moveETHGainToTrove{ value: depositorETHGain }(msg.sender, _upperHint, _lowerHint);
-    }
-
-    // --- ZERO issuance functions ---
-
-    function _triggerZEROIssuance(ICommunityIssuance _communityIssuance) internal {
-        uint ZEROIssuance = _communityIssuance.issueZERO();
-       _updateG(ZEROIssuance);
-    }
-
-    function _updateG(uint _ZEROIssuance) internal {
-        uint totalZUSD = totalZUSDDeposits; // cached to save an SLOAD
-        /*
-        * When total deposits is 0, G is not updated. In this case, the ZERO issued can not be obtained by later
-        * depositors - it is missed out on, and remains in the balanceof the CommunityIssuance contract.
-        *
-        */
-        if (totalZUSD == 0 || _ZEROIssuance == 0) {return;}
-
-        uint ZEROPerUnitStaked;
-        ZEROPerUnitStaked =_computeZEROPerUnitStaked(_ZEROIssuance, totalZUSD);
-
-        uint marginalZEROGain = ZEROPerUnitStaked.mul(P);
-        epochToScaleToG[currentEpoch][currentScale] = epochToScaleToG[currentEpoch][currentScale].add(marginalZEROGain);
+        emit StabilityPoolSOVBalanceUpdated(sovToken.balanceOf(address(this)));
+        emit SOVSent(msg.sender, depositorSOVGain);
 
-        emit G_Updated(epochToScaleToG[currentEpoch][currentScale], currentEpoch, currentScale);
-    }
-
-    function _computeZEROPerUnitStaked(uint _ZEROIssuance, uint _totalZUSDDeposits) internal returns (uint) {
-        /*  
-        * Calculate the ZERO-per-unit staked.  Division uses a "feedback" error correction, to keep the 
-        * cumulative error low in the running total G:
-        *
-        * 1) Form a numerator which compensates for the floor division error that occurred the last time this 
-        * function was called.  
-        * 2) Calculate "per-unit-staked" ratio.
-        * 3) Multiply the ratio back by its denominator, to reveal the current floor division error.
-        * 4) Store this error for use in the next correction when this function is called.
-        * 5) Note: static analysis tools complain about this "division before multiplication", however, it is intended.
-        */
-        uint ZERONumerator = _ZEROIssuance.mul(DECIMAL_PRECISION).add(lastZEROError);
-
-        uint ZEROPerUnitStaked = ZERONumerator.div(_totalZUSDDeposits);
-        lastZEROError = ZERONumerator.sub(ZEROPerUnitStaked.mul(_totalZUSDDeposits));
-
-        return ZEROPerUnitStaked;
```

Comment:
Why have you deleted this?

Reply:
Becasue stabilityPool contract doesn't issue ZERO tokens